app/containers/UserCalendar/index.js

```diff
@@ -30,14 +30,14 @@ const UserCalendar = ({ loading, viewer, user, router, refetch }) => {
 
   return (
     <>
-      <Head>
-        <title>Calendar | CodingBlocks</title>
-      </Head>
-      <AppContent>
-        <Content user={selectedUser} />
-      </AppContent>
-      {['Admin', 'Member'].includes(viewers.user.role) && (
+      {authHelper.isMember(viewers) && (
         <>
+          <Head>
+            <title>Calendar | CodingBlocks</title>
+          </Head>
+          <AppContent>
+            <Content user={selectedUser} />
+          </AppContent>
```

Comment:
You just need to stop wrapping the Content with AppContent if the viewer is not a member.

Reply:
Thats I have done at line 56.

Reply:
We should not duplicate the whole bunch of code. 

Just the Content part. Also if the user is not a member, wrap the Content with <div class="row">